doto-frontend/src/components/pages/Calendar/__snapshots__/Calendar.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<Calendar /> component being rendered Make sure render matches snapshot 1`] = `ReactWrapper {}`;
```

Comment:
I think this test is not rendering properly, it's suppsoed to render the whole calendar component

Reply:
Oh no I forgot to do toJson() the component. Will make the change soon

Reply:
Ok, there is a problem using renderer like other test files. renderer from "react-test-renderer";
There is an error saying TypeError: Cannot read property 'querySelector' of null which doesn't let me run renderer.create()
Mounting using enzyme has no json but instead can do debug() which returns snapshot in strings which is enough for snapshot testing. However, using mount Object with debug(), it creates 4000+ lines long .snap file. Git does not let me upload this due to its size and also has very long test time.

SO, I've used shallow from enzyme to do snapshot testing which doesn't really have much information but I guess it is better than nothing. (or is it?)